{
    "@timestamp": "2022-03-03T10:42:25.842Z",
    "agent": {
        "ephemeral_id": "ee8c5c05-6c17-476c-af50-7e1aabb3d98d",
        "id": "f81b0fc1-f2c5-45e6-8f5d-2e969313b9b4",
        "name": "docker-fleet-agent",
        "type": "filebeat",
        "version": "8.0.0"
    },
    "cisco_ise": {
        "log": {
            "ad": {
                "admin": "ise.host.local",
                "domain": {
                    "controller": "host.local",
                    "name": "host.local"
                },
                "error": {
                    "details": "The user account is invalid"
                },
                "forest": "host.local",
                "hostname": "cisco-ise-host@host.local",
                "ip": "89.160.20.156",
                "log_id": "1645524126/47",
                "site": "Default-First-Site-Name"
            },
            "category": {
                "name": "CISE_AD_Connector"
            },
            "log_details": "AD-Admin=ise.host.local, AD-Domain=host.local, AD-Domain-Controller=host.local, AD-Error-Details=The user account is invalid, AD-Forest=host.local, AD-Hostname=cisco-ise-host@host.local, AD-IP-Address=89.160.20.156, AD-Log-Id=1645524126/47, AD-Organization-Unit=, AD-Site=Default-First-Site-Name",
            "message": {
                "code": "25012",
                "description": "AD-Connector: Domain join failed",
                "id": "0000083094"
            },
            "segment": {
                "number": 0,
                "total": 1
            },
            "sequence_number": "0000083161"
        }
    },
    "data_stream": {
        "dataset": "cisco_ise.log",
        "namespace": "ep",
        "type": "logs"
    },
    "ecs": {
        "version": "8.0.0"
    },
    "elastic_agent": {
        "id": "f81b0fc1-f2c5-45e6-8f5d-2e969313b9b4",
        "snapshot": false,
        "version": "8.0.0"
    },
    "event": {
        "action": "ad-connector",
        "agent_id_status": "verified",
        "category": [
            "authentication"
        ],
        "dataset": "cisco_ise.log",
        "ingested": "2022-03-15T10:03:12Z",
        "kind": "event",
        "timezone": "+00:00",
        "type": [
            "info",
            "end"
        ]
    },
    "host": {
        "hostname": "cisco-ise-host"
    },
    "input": {
        "type": "tcp"
    },
    "log": {
        "level": "warn",
        "source": {
            "address": "192.168.112.6:52648"
        },
        "syslog": {
            "priority": 180,
            "severity": {
                "name": "warn"
            }
        }
    },
    "related": {
        "hosts": [
            "cisco-ise-host",
            "cisco-ise-host@host.local"
        ],
        "ip": [
            "89.160.20.156"
        ]
    },
    "tags": [
        "forwarded",
        "cisco_ise-log"
    ]
}